src/index.js

```diff
@@ -1,18 +1,23 @@
+import {Ion, Viewer, createWorldTerrain, createOsmBuildings, Cartesian3, Math } from "../node_modules/cesium"
```

Comment:
```suggestion
import { Ion, Viewer, createWorldTerrain, createOsmBuildings, Cartesian3, Math } from "../node_modules/cesium"
```

Reply:
Did you have any luck getting this to work with the cesium alias, ie `from "cesium"` ?